src/psyclone/tests/psyir/transformations/loop_tiling_2d_trans_test.py

```diff
@@ -164,3 +190,38 @@ def test_loop_tiling_2d_trans_apply(fortran_reader, fortran_writer):
   enddo
 enddo'''
     assert expected in result
+
+
+def test_loop_tiling_2d_trans_apply_options(fortran_reader, fortran_writer):
+    ''' Validation passes when found a 2D nested loop construct when a
```

Comment:
Maybe "Check that a non-default tilesize option is used correctly"?

Reply:
Done